internal/db/schema/migrations/oss/postgres/29/02_kms_version.up.sql

```diff
@@ -0,0 +1,38 @@
+begin;
+
+-- make the required schema changes to adopt
+-- github.com/hashicorp/go-kms-wrapping/extras/kms/v2 
+-- this migration is from:
+-- https://github.com/hashicorp/go-kms-wrapping/blob/main/extras/kms/migrations/postgres/02_version.up.sql 
+
+create table kms_schema_version(
+    version text not null,
+    create_time kms_timestamp,
+    update_time kms_timestamp
+);
+
+-- this index ensures that there will only ever be one row in the 
+-- table.  see:
+-- https://www.postgresql.org/docs/current/indexes-expressional.html
+create unique index kms_schema_version_one_row
+ON kms_schema_version((version is not null));
+
+ -- define the immutable fields for kms_root_key (all of them)
+create trigger kms_immutable_columns
+before
+update on kms_schema_version
+  for each row execute procedure kms_immutable_columns('create_time');
+
+create trigger kms_default_create_time_column
+before
+insert on kms_schema_version
+  for each row execute procedure kms_default_create_time();
+
+create trigger update_time_column 
+before 
+update on kms_schema_version 
+	for each row execute procedure update_time_column();
```

Comment:
Should this be calling `kms_update_time_column`?

Reply:
Hmm it looks like this is called `update_time_column` in [kms 01_domain_types](https://github.com/hashicorp/go-kms-wrapping/blob/a4371f584490f016b59576c1b765af1446c97d7a/extras/kms/migrations/postgres/01_domain_types.up.sql#L76-L94). I think ideally that would be `kms_update_time_column` to avoid the overlap, but idk how easily it can be changed now...

Reply:
so.. we should change it... but it will make comparing diffs a bit more difficult.  I need to open an upstream PR with a change migration.

Reply:
I know of only one other usage... so we can probably just fix the migration?

Reply:
working on a proper upstream PR with a new migration.

Reply:
done!